modules/swagger-codegen/src/main/resources/nodejs/writer.mustache

```diff
@@ -7,9 +7,10 @@ exports.respondWithCode = function(code, payload) {
   return new ResponsePayload(code, payload);
 }
 
-var writeJson = exports.writeJson = function(response, arg1, arg2) {
+var writeJson = exports.writeJson = function(response, arg1, arg2, arg3) {
```

Comment:
@sandro97git Shall we have better naming for arg1, arg2, arg3?

Reply:
I think this makes sense. What do you suggest?